docs/api-docs/provider-apis/tax/overview.md

```diff
@@ -58,19 +58,42 @@ Once your tax provider configuration is ready, we'll let you know via email. The
 | **Testing**                               |                                |                              |                                                                                                       |                                                   |
 | Partner test store domain(s)          | Required                       | Domain name                  | Share your test store domain(s) sow we can test prior to launching your tax provider.                 | `https://sampletax-test-store.mybigcommerce.com/` |
 
+### Sandbox tax provider configuration
+
+Sandbox tax provider configurations are nested within your production tax provider configuration, as such tax providers requiring a sandbox tax provider configuration should also provide the details for their production tax provider configuration.
+
+Additionally, when providing the details for a sandbox tax provider configuration, it's unncessary to specify the below properties because they are inherited from the parent production tax provider configuration (all other properties are still required):
+
+* Supported store(s)
+* Supported store hash(es) (in the case tax provider is a private instance)
```

Comment:
What are the differences here?

Reply:
Thinking about it some more, given in the code **support_stores** is associated to store hash values and when this is blank we assume all stores are supported, the above is a bit duplicative. Would it make more sense if I remove the first line, "Supported store(s)" and leave the second?

Reply:
I think so

Reply:
Updated [here](1e09ba04983591550d8f744afb960444d220c2bb), thoughts?

Reply:
Lovely, thanks @bc-rogq.